gdp-src-build/conf/templates/intel_nuc-core-i5.local.conf

```diff
@@ -11,5 +11,5 @@ PREFERRED_PROVIDER_virtual/mesa = ""
 PREFERRED_PROVIDER_libgbm = "libgbm"
 PREFERRED_PROVIDER_libgbm-dev = "libgbm"
 SDKIMAGE_FEATURES_append = " staticdev-pkgs"
-IMAGE_INSTALL_append = " mesa-megadriver"
+IMAGE_INSTALL_append = " mesa-megadriver qtbase rtl-sdr radioservice alsajackout fmradioqtplugin gstreamer1.0-plugins-good"
```

Comment:
or maybe you should fix dependencies in your fmradio.bbappend and others in your layer ?